### DISCUSSION TITLE
Cant run microservices

### DISCUSSION THREAD

**Discussion by msafwa87** [socioboard/Socioboard-5.0#370] [General]
Hello
I have been trying to run microservices. I am currently using the instruction from https://www.linuxbabe.com/ubuntu/install-socioboard-on-ubuntu-20-04 so i have been trying to start the microservices and getting some errors as attached
![socioboard_error1](https://user-images.githubusercontent.com/65029572/171414579-9f32fa93-5544-4c5c-bf85-0f60f4fe6d76.JPG)

**Comment by TechnologyClassroom** [comment]
Read this issue where people are figuring it out: https://github.com/socioboard/Socioboard-5.0/issues/112

That guide is for version 4.

**Reply by msafwa87** [reply]
Thanks

**Comment by msafwa87** [comment]
I am also getting this error when i run "npm run swagger" in socioboard-api/User folder
![socio_error1](https://user-images.githubusercontent.com/65029572/171563478-daff779d-3248-4dca-a0ee-41266ecd3d9f.JPG)

**Reply by jon4short** [reply]
Use my install script. It fixes a majority of the issues. And then add the entry mentioned in this thread to the config file.

**Reply by msafwa87** [reply]
Sorry i am a newbie in Linux, can you please specifically tell me what i should exactly do.

**Reply by msafwa87** [reply]
Hi Jon4short, wanted to give you an update, i managed to start some of the microservices but when i run nodemon update.server.js
I get below error:-
 error: Error: Configuration property "test_url" is not defined
      at Config.get (/var/www/socioboard/socioboard-api/Common/node_modules/config/lib/config.js:182:11)
      at getEnvStrict (file:///var/www/socioboard/socioboard-api/Common/Shared/config.shared.js:3:45)
      at file:///var/www/socioboard/socioboard-api/Common/Constants/tiktok.constants.js:6:22
      at ModuleJob.run (internal/modules/esm/module_job.js:183:25)
      at async Loader.import (internal/modules/esm/loader.js:178:24)
      at async Object.loadESM (internal/process/esm_loader.js:68:5)
      at async handleMainPromise (internal/modules/run_main.js:59:12),
  level: 'error',
